test/static-code

```diff
@@ -45,13 +45,21 @@ test_ruff() {
 }
 
 if [ "${WITH_PARTIAL_TREE:-0}" = 0 ]; then
+    test_mypy() {
+        for pkg in systemd_ctypes ferny bei; do
+            test -e "src/cockpit/_vendor/${pkg}/__init__.py" || skip "no ${pkg}"
+        done
+        mypy --no-error-summary src/cockpit test/pytest
```

Comment:
That needs a `command -v` test similar to vulture?

Reply:
Thanks!